tests/taskcluster/bin/raptor_graph

```diff
@@ -52,36 +63,70 @@ if testing a PULL_REQUEST 'patch build' will be the pr/
 provided HEAD_REV; else it will be BASE_REV -1 commit
 */
 
-var config = {
-  baseTask1: {
-    definition: 'raptor_launch',
-    params: {
-      raptorSuiteName: 'Settings Launch Test (on base)',
-      gaiaRev: process.env.GITHUB_BASE_REV,
-      raptorRuns: 30,
-      raptorApps: 'settings',
-      treeherderSym: 'se1'
-    }
-  },
-  patchTask1: {
-    definition: 'raptor_launch',
-    params: {
-      raptorSuiteName: 'Settings Launch Test (on base + pr)',
-      gaiaRev: process.env.GITHUB_HEAD_REV,
-      raptorRuns: 30,
-      raptorApps: 'settings',
-      treeherderSym: 'se2'
-    }
-  },
-  resultTask: {
+// build the task config object for each app
+// each app will have 1 base task and 1 patch task x TEST_ITERATIONS,
+// and there is always just one result task (per app) that is
+// dependent upon all the base and patch tasks completed
+var config = {};
+
+LAUNCH_TEST_APPS.map(function(nextApp) {
+  // results task requires all base and patch tasks
+  requiredBaseTasks = [];
+  requiredPatchTasks = [];
+
+  // base tasks
+  for (var iteration = 1; iteration <= TEST_ITERATIONS; iteration++) {
+    suiteName = nextApp[0].toUpperCase() + nextApp.slice(1) + " app launch test (on base)";
+    symbol = nextApp.substr(0, 2) + iteration;
```

Comment:
We decided the best symbol to use for now was the first two letters of each app, and there are no conflicting apps at the moment that we will run the test on, but possibly down the road we'll have to override the symbol if an app name conflicts.